libraries/core/src/test/java/com/paritytrading/philadelphia/FIXInitiatorPipeTest.java

```diff
@@ -0,0 +1,24 @@
+package com.paritytrading.philadelphia;
+
+import java.io.IOException;
+import java.nio.channels.Pipe;
+import org.jetbrains.annotations.NotNull;
+
+public class FIXInitiatorPipeTest extends FIXInitiatorTest<DoublePipeChannel> {
```

Comment:
What do you think if we called this `FIXInitiatorTestWithPipeChannel`?

Let's make also this class package-private.

Reply:
Some tools (especially those desighed in JUnit 3 timeframe) expect test classes to finish with `*Test`. The IntelliJ IDEA `Class naming convention` inspection is one of these.

Many people have similar expectations :-)

Renamed.